biodatasets/bio_simlex/bio_simlex.py

```diff
@@ -0,0 +1,181 @@
+[bigbio_schema_name] = pairs
+"""
+
+from typing import Dict, List, Tuple
+
+import datasets
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+# TODO: Add BibTeX citation
+_CITATION = """\
+@article{article,
+author = {Chiu, Billy and Pyysalo, Sampo and Vulić, Ivan and Korhonen, Anna},
+year = {2018},
+month = {02},
+pages = {},
+title = {Bio-SimVerb and Bio-SimLex: Wide-coverage evaluation sets of word similarity in biomedicine},
+volume = {19},
+journal = {BMC Bioinformatics},
+doi = {10.1186/s12859-018-2039-z}
+}{}
+}
+"""
+
+_DATASETNAME = "bio_simlex"
+
+_DESCRIPTION = """\
+Bio-SimLex enables intrinsic evaluation of word representations. This evaluation can serve as a predictor of
+performance on various downstream tasks in the biomedical domain. The results on Bio-SimLex using standard
+word representation models highlight the importance of developing dedicated evaluation resources for NLP in biomedicine
+for particular word classes (e.g. verbs).
+"""
+
+_HOMEPAGE = "https://github.com/cambridgeltl/bio-simverb"
+
+
+_LICENSE = """Open Access This article is distributed under the terms of the
+Creative Commons Attribution 4.0 International License
+(http://creativecommons.org/licenses/by/4.0/), which permits
+unrestricted use, distribution, and reproduction in any medium,
+provided you give appropriate credit to the original author(s) and
+the source, provide a link to the Creative Commons license, and
+indicate if changes were made. The Creative Commons Public Domain
+Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/)
+applies to the data made available in this article, unless otherwise stated."""
+
+
+_URLS = {
+    _DATASETNAME: "https://github.com/cambridgeltl/bio-simverb/blob/master/wvlib/word-similarities/\
+bio-simlex/Bio-SimLex.txt?raw=true"
+}
+
+_SUPPORTED_TASKS = [Tasks.SEMANTIC_SIMILARITY]
+
+_SOURCE_VERSION = "1.0.0"
+
+_BIGBIO_VERSION = "1.0.0"
+
+
+class BioSimlexDataset(datasets.GeneratorBasedBuilder):
+    """Bio-SimLex enables intrinsic evaluation of word representations. Config schema as source gives score between
+    0-10 for pairs of words. Config schema as bigbio gives label 0 and 1 for pairs of words."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    # You will be able to load the "source" or "bigbio" configurations with
+    # ds_source = datasets.load_dataset('my_dataset', name='source')
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio')
+
+    # For local datasets you can make use of the `data_dir` and `data_files` kwargs
+    # https://huggingface.co/docs/datasets/add_dataset.html#downloading-data-files-and-organizing-splits
+    # ds_source = datasets.load_dataset('my_dataset', name='source', data_dir="/path/to/data/files")
+    # ds_bigbio = datasets.load_dataset('my_dataset', name='bigbio', data_dir="/path/to/data/files")
```

Comment:
can we remove these comments (line 89-96)? they are generic templates and wont apply for this particular dataset.

Reply:
done